src/variables/components/TypeAheadVariableDropdown.tsx

```diff
@@ -278,11 +378,27 @@ class TypeAheadVariableDropdown extends PureComponent<Props, MyState> {
   // when it is loading
   getPlaceHolderText = (defaultText: string = '') => {
     const {status} = this.props
+
     if (status === RemoteDataState.Loading) {
       return 'Loading...'
     }
+    if (this.noValuesPresent(false)) {
+      return 'No Values'
+    }
+
     return defaultText
   }
+
+  noValuesPresent = useAllValues => {
```

Comment:
boolean arguments are usually a smell that this should just be two functions https://martinfowler.com/bliki/FlagArgument.html

what do you think?

Reply:
I separated them but kept a hidden internal method to avoid code duplication